### PR TITLE
Docs/clarinet js sdk updates

### PR DESCRIPTION
**Pull request by ryanwaits**
Some copy changes, code formatting, and general feedback updates based on [PR #531](https://github.com/hirosystems/docs/pull/531#pullrequestreview-1660561930)

### CODE REVIEW

File: docs/feature-guides/clarinet-js-sdk.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-The Clarinet SDK is used to spawn and interact with a Simnet from Node.js.
+The Clarinet SDK is a JavaScript library to spawn and interact with a simulated Clarinet environment, also known as simnet.
 
 A Simnet is a simulated network that simulates the Stacks blockchain and runs the Clarity VM, without the need for actual Stacks and Bitcoin nodes (unlike Devnet, Testnet and Mainnet).
 
 Here is a non-exhaustive list of some of the use-cass:
+
 - Call public and read-only functions from smart contracts
 - Get clarity maps or data-var values
 - Get contract ABI
```
Comment (sabbyanandan) [inline docs/feature-guides/clarinet-js-sdk.md:11]: The references to "ABI" may not be a common convention to some developers. Perhaps worth expanding on the acronym somewhere in the page.

File: docs/feature-guides/clarinet-js-sdk.md [hunk 2]
```diff
@@ -414,7 +425,7 @@ getContractsInterfaces(): Map<string, ContractInterface>
 
 Returns the interfaces of the project contracts.
 It returns a Map of Contracts, the keys are the contract addresses.
-The interfaces contains informations such as the ABI, the available functions, data-vars and maps, the NFT and FT defined in the contract  .
+The interfaces contains informations such as the ABI, the available functions, data-vars and maps, the NFT and FT defined in the contract .
```
Comment (sabbyanandan) [inline docs/feature-guides/clarinet-js-sdk.md:428]:
```suggestion
The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFT, and the FTs defined in the contract.
```

File: docs/feature-guides/clarinet-js-sdk.md [hunk 3]
```diff
@@ -414,7 +425,7 @@ getContractsInterfaces(): Map<string, ContractInterface>
 
 Returns the interfaces of the project contracts.
```
Comment (sabbyanandan) [inline docs/feature-guides/clarinet-js-sdk.md:426]: Some parts of this language can be revisited (in a different PR, perhaps) for tonality. cc: @max-crawford

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
A few nits, @ryanwaits. And something to consider in a different PR. Overall, looks good.

File: docs/feature-guides/clarinet-js-sdk.md [hunk 4]
```diff
@@ -414,7 +425,7 @@ getContractsInterfaces(): Map<string, ContractInterface>
...
 It returns a Map of Contracts, the keys are the contract addresses.
-The interfaces contains informations such as the ABI, the available functions, data-vars and maps, the NFT and FT defined in the contract  .
+The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFT, and the FTs defined in the contract.
```
Comment (hugocaillard) [inline docs/feature-guides/clarinet-js-sdk.md:428]:
```suggestion
The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.
```

File: docs/feature-guides/clarinet-js-sdk.md (hunk 1 above)
Comment (hugocaillard) [inline docs/feature-guides/clarinet-js-sdk.md:11]:
```suggestion
- Get contract interfaces (available functions and data)
```

File: docs/feature-guides/clarinet-js-sdk.md [hunk 5]
```diff
@@ -414,7 +425,7 @@ getContractsInterfaces(): Map<string, ContractInterface>
 
 Returns the interfaces of the project contracts.
 It returns a Map of Contracts, the keys are the contract addresses.
-The interfaces contains informations such as the ABI, the available functions, data-vars and maps, the NFT and FT defined in the contract  .
+The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.
```
Comment (hugocaillard) [inline docs/feature-guides/clarinet-js-sdk.md:428]:
```suggestion
The interfaces contain information such as the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.
```

File: docs/feature-guides/clarinet-js-sdk.md (hunk 1 above)
Comment (hugocaillard) [reply docs/feature-guides/clarinet-js-sdk.md:11]: Made two suggestions too fully remove ABI. I don't it's 100% technically correct anyway

File: docs/feature-guides/clarinet-js-sdk.md (hunk 3 above)
Comment (hugocaillard) [reply docs/feature-guides/clarinet-js-sdk.md:426]: @sabbyanandan 
Resolving, we need to merge this PR to fix docs build

**Review by hugocaillard — APPROVED** [review]

**Pull request merged**